doc/fluid/design/onnx/onnx_convertor.md

```diff
@@ -0,0 +1,37 @@
+### Backgroud
+
+(@kuke)
+
+### How it works
+(@kuke)
+
+### Project structure
+
+<p align="center">
+<img src="./images/project_structure.png"/>
+</p>
+
+The project contains four important modules: 
+
+* **fluid**: Contain wrappers for fluid related APIs. Fluid has provided some low-level APIs to parse or generate the inference model. However, directly using these low-level APIs makes the code tediously long. This module wraps low-level APIs to provide simplied interfaces.
+
+* **onnx**: ONNX uses proto file to save computation flow and model weights. This module is responsible for parsing and generating ONNX binary model.
+
+* **onnx_fluid**: Concepts in fluid like program, block etc. haven't direct corresponding concepts in ONNX. Even that both contains operator concept, for many operators adaption is also necessary. This module is the most important module responsible for acutal converting. Adaption for different level concepts should be provided like fluid program/block to ONNX graph, fluid operators to ONNX operators etc.
```

Comment:
This should be as follows:

Concepts in fluid like program, block etc. **do not have** any direct corresponding concepts in ONNX. **Even though both contain the operator concept**, for many operators **adaptation** is also necessary. This module is the most important module **and is** responsible for **the actual** **conversion**. Adaption for different level **of** concepts should be provided like fluid program/block to ONNX graph, fluid operators to ONNX operators etc.